packages/icons/brands/Discord.tsx

```diff
@@ -0,0 +1,9 @@
+import { createIcon } from '../utils'
+
+export const DiscordIcon = createIcon(
+    'Discord',
+    <>
+        <path d="M395 151s-40-31-88-35l-4 8c43 11 63 26 83 44-35-18-70-35-131-35s-96 17-131 35c20-18 44-35 83-44l-5-8c-49 5-87 35-87 35s-45 65-53 193c46 52 114 52 114 52l15-19c-25-8-52-24-76-51 28 21 71 44 140 44s112-22 140-44c-23 27-51 43-75 51l14 19s69 0 114-52c-8-128-53-193-53-193zM198 309c-17 0-31-16-31-35s14-36 31-36 31 16 31 36-14 35-31 35zm114 0c-17 0-31-16-31-35s14-36 31-36 31 16 31 36-14 35-31 35z" />
```

Comment:
@Jack-Works we may be need svg to tsx code-generator or webpack-loader.
CC @guanbinrui

Reply:
Yes. I prefer codegen

Reply:
@Jack-Works plz create a issue, and design a solution.

Reply:
#3563